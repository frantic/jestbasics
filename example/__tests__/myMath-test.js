jest.dontMock('../myMath');

const myMath = require('../myMath');
const fetch = require('fetch');

describe('my math module', () => {
  // beforeAll(() => {
  //   console.log('This is executed before the test suite');
  // });

  // beforeEach(() => {
  //   console.log('This is executed before each testcase');
  // });

  it('adds two numbers', () => {
    expect(myMath.add(1, 1)).toEqual(2);
  });

  it('gets random number', () => {
    return myMath.randomNumber().then((number) => {
      expect(number).toEqual(42);
      expect(fetch).toBeCalled();
    });
  })
});
