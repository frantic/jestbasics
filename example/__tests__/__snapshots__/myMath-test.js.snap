exports[`my math module records snapshots 1`] = `
Object {
  "foo": 1
}
`;
